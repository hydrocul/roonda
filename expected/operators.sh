perl $ROONDA_TMP_PATH/roonda_fb27e23d5e8e86fc19e2b2172c8067dd7f11fa8e.pl
ruby $ROONDA_TMP_PATH/roonda_eb4500cde25eebb81e89caa887408212424a84d5.rb
python2 $ROONDA_TMP_PATH/roonda_6b976a1201489edc0d73a9a0b34c1e4bb0fff88f.py
php $ROONDA_TMP_PATH/roonda_97c0e9e3497ac56e4f698afefb2241f0e318a1be.php

#################################################
# roonda_fb27e23d5e8e86fc19e2b2172c8067dd7f11fa8e.pl:
#################################################
# print 3 * 4 * (5 + 6);
# print "\n";
# print "abc" . "def" . "\n";

#################################################
# roonda_eb4500cde25eebb81e89caa887408212424a84d5.rb:
#################################################
# print 3 * 4 * (5 + 6)
# print "\n"
# print "abc" + "def" + "\n"

#################################################
# roonda_6b976a1201489edc0d73a9a0b34c1e4bb0fff88f.py:
#################################################
# import sys
# 
# sys.stdout.write(str(3 * 4 * (5 + 6)))
# sys.stdout.write(str("\n"))
# sys.stdout.write(str("abc" + "def" + "\n"))

#################################################
# roonda_97c0e9e3497ac56e4f698afefb2241f0e318a1be.php:
#################################################
# <?php
# 
# echo 3 * 4 * (5 + 6);
# echo "\n";
# echo "abc" . "def" . "\n";
