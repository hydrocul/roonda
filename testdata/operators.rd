(sh v1)

((perl)
 (print (* 3 4 (+ 5 6))) (print "\n")
 (print (% 100 7)) (print "\n")
 (print (** 3 5)) (print "\n")
 (print (strcat "abc" "def" "\n")))

((ruby)
 (print (* 3 4 (+ 5 6))) (print "\n")
 (print (% 100 7)) (print "\n")
 (print (** 3 5)) (print "\n")
 (print (strcat "abc" "def" "\n")))

((python2)
 (print (* 3 4 (+ 5 6))) (print "\n")
 (print (% 100 7)) (print "\n")
 (print (** 3 5)) (print "\n")
 (print (strcat "abc" "def" "\n")))

((python3)
 (print (* 3 4 (+ 5 6))) (print "\n")
 (print (% 100 7)) (print "\n")
 (print (** 3 5)) (print "\n")
 (print (strcat "abc" "def" "\n")))

((php)
 (print (* 3 4 (+ 5 6))) (print "\n")
 (print (% 100 7)) (print "\n")
 (print (** 3 5)) (print "\n")
 (print (strcat "abc" "def" "\n")))

