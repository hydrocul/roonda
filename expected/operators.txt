132
abcdef
132
abcdef
132
abcdef
132
abcdef
